packages/openapi-pinia-colada/test/fixtures/openapi-typescript-codegen.min.js

```diff
@@ -0,0 +1,10 @@
rg:r},body:e,mediaType:"application/json",errors:{404:"Resource not found",422:"Validation failed, or the endpoint has been spammed."}})}static orgsCancelInvitation(r,e){return o(i,{method:"DELETE",url:"/orgs/{org}/invitations/{invitation_id}",path:{org:r,invitation_id:e},errors:{404:"Resource not found",422:"Validation failed, or the endpoint has been spammed."}})}static orgsListInvitationTeams(r,e,n=30,s=1){return o(i,{method:"GET",url:"/orgs/{org}/invitations/{invitation_id}/teams",path:{org:r,invitation_id:e},query:{per_page:n,page:s},errors:{404:"Resource not found"}})}static orgsListMembers(r,e="all",n="all",s=30,u=1){return o(i,{method:"GET",url:"/orgs/{org}/members",path:{org:r},query:{filter:e,role:n,per_page:s,page:u},errors:{422:"Validation failed, or the endpoint has been spammed."}})}static orgsCheckMembershipForUser(r,e){return o(i,{method:"GET",url:"/orgs/{org}/members/{username}",path:{org:r,username:e},errors:{302:"Response if requester is not an organization member",404:"Not Found if requester is an organization member and user is not a member"}})}static orgsRemoveMember(r,e){return o(i,{method:"DELETE",url:"/orgs/{org}/members/{username}",path:{org:r,username:e},errors:{403:"Forbidden"}})}static orgsGetMembershipForUser(r,e){return o(i,{method:"GET",url:"/orgs/{org}/memberships/{username}",path:{org:r,username:e},errors:{403:"Forbidden",404:"Resource not found"}})}static orgsSetMembershipForUser(r,e,n){return o(i,{method:"PUT",url:"/orgs/{org}/memberships/{username}",path:{org:r,username:e},body:n,mediaType:"application/json",errors:{403:"Forbidden",422:"Validation failed, or the endpoint has been spammed."}})}static orgsRemoveMembershipForUser(r,e){return o(i,{method:"DELETE",url:"/orgs/{org}/memberships/{username}",path:{org:r,username:e},errors:{403:"Forbidden",404:"Resource not found"}})}static orgsListOutsideCollaborators(r,e="all",n=30,s=1){return o(i,{method:"GET",url:"/orgs/{org}/outside_collaborators",path:{org:r},query:{filter:e,per_page:n,page:s}})}static orgsConvertMemberToOutsideCollaborator(r,e,n){return o(i,{method:"PUT",url:"/orgs/{org}/outside_collaborators/{username}",path:{org:r,username:e},body:n,mediaType:"application/json",errors:{403:'Forbidden if user is the last owner of the organization, not a member of the organization, or if the enterprise enforces a policy for inviting outside collaborators. For more information, see "[Enforcing repository management policies in your enterprise](https://docs.github.com/admin/policies/enforcing-policies-for-your-enterprise/enforcing-repository-management-policies-in-your-enterprise#enforcing-a-policy-for-inviting-outside-collaborators-to-repositories)."',404:"Resource not found"}})}static orgsRemoveOutsideCollaborator(r,e){return o(i,{method:"DELETE",url:"/orgs/{org}/outside_collaborators/{username}",path:{org:r,username:e},errors:{422:"Unprocessable Entity if user is a member of the organization"}})}static orgsListPatGrantRequests(r,e=30,n=1,s="created_at",u="desc",
```

Comment:
This isn’t really needed; this is a third-party library we benchmark against openapi-fetch. We only build it and commit it so we’re testing the actual runtime version and don’t unnecessarily report slower numbers from JIT compilation, etc.